source/dmocks/method_mock.d

```diff
@@ -61,11 +61,11 @@ string BuildForwardCall(string methodName, bool dynamicVarArgs)()
     enum methodString = dynamicVarArgs ? "v"~methodName : methodName;
     enum argsPassed = dynamicVarArgs ? "(args, varArgsList, varArgsPtr)" : "(args)";
 
-    return `static if (is (typeof (mocked___.` ~ methodString~`)))
+    return `static if (is (typeof (mocked___.` ~ methodString ~ argsPassed~`)))
```

Comment:
consequent whitespaces around `~`.
Any way to unittest this?

Reply:
It's a workaround for a weird compiler bug that I don't have a proper reduction for.

Reply:
Whitespaces fixed